distribution/solr-distro/assembly.xml

```diff
@@ -42,20 +42,29 @@
         </dependencySet>
     </dependencySets>
     <fileSets>
+        <!-- The order of the <fileSet> elements is important. The assumbly plugin will not
+        replace an existing file. Therefore, files that override behavior in the Apache Solr
+        distribution must be come before the Apache Solr files themselves. -->
+
+        <!-- Files with executable permissions -->
         <fileSet>
             <directory>src/main/resources
             </directory>
             <outputDirectory>/</outputDirectory>
-            <!--These changes are included to turn on Pro-Grade Security Manager for Solr-->
             <includes>
-                <include>security/solr-default.policy</include>
-                <include>bin/solr</include>
-                <!--This can be changed once Solr is upgraded to 7.5/8.0-->
-                <!--https://issues.apache.org/jira/browse/SOLR-12538-->
-                <include>bin/solr.cmd</include>
+                <include>bin/*</include>
             </includes>
-            <fileMode>0755</fileMode>
+            <fileMode>0754</fileMode>
```

Comment:
❓ Why only read for other? Either set it to 0 or set it to 5 so they can execute too.

Reply:
set to 0